src/components/FormQuestions.tsx

```
@@ -190,10 +197,36 @@ const FormQuestions = ({ event, errorHandler }: FormQuestionProp) => {
+                      name,
+                      studentId,
+                      phone,
+                    },
+                  });
+
+                  if (result?.isParticipable) {
+                    setPageNum((prev) => (prev += 1));
+                  } else if (result?.errorCodeName) {
+                    errorHandler(result.errorCodeName as ErrorCodeType);
+                  }
+                } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)